pmd-core/src/main/java/net/sourceforge/pmd/renderers/CodeClimateRenderer.java

```diff
@@ -36,18 +37,17 @@
     // Note: required by https://github.com/codeclimate/spec/blob/master/SPEC.md
     protected static final String NULL_CHARACTER = "\u0000";
     protected static final List<String> INTERNAL_DEV_PROPERTIES = Arrays.asList("version", "xpath");
-    private final String pmdDeveloperUrl;
+    private static final String PMD_PROPERTIES_URL = getPmdPropertiesURL();
     private Rule rule;
 
     public CodeClimateRenderer() {
         super(NAME, "Code Climate integration.");
-        pmdDeveloperUrl = getPmdDeveloperURL();
     }
 
-    private static String getPmdDeveloperURL() {
-        String url = "http://pmd.github.io/pmd-" + PMD.VERSION + "/customizing/pmd-developer.html";
-        if (PMD.VERSION.contains("SNAPSHOT") || "unknown".equals(PMD.VERSION)) {
-            url = "http://pmd.sourceforge.net/snapshot/customizing/pmd-developer.html";
+    private static String getPmdPropertiesURL() {
+        String url = "https://pmd.github.io/pmd-" + PMDVersion.VERSION + "/pmd_devdocs_working_with_properties.html";
+        if (PMDVersion.isSnapshot() || PMDVersion.isUnknown()) {
+            url = "https://pmd.github.io/latest/pmd_devdocs_working_with_properties.html";
```

Comment:
We should set up a reminder to update that when the new version of the site goes online.